src/App.js

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [chatLogEntries, setChatLogEntries] = useState(chatMessages);
+
+  const updateChatEntry = (updatedChatEntry) => {
+    setChatLogEntries(
+      chatLogEntries.map((entry) =>
+        entry.id === updatedChatEntry.id ? updatedChatEntry : entry
+      )
+    );
+  };
+
+  const [localUser, ...remoteUsers] = [
+    ...new Set(chatLogEntries.map((entry) => entry.sender)),
+  ];
+  const totalLikes = chatLogEntries.reduce(
+    (count, message) => count + (message.liked ? 1 : 0),
+    0
+  );
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>
+          Chat between {localUser} and {remoteUsers}
```

Comment:
So I was intrigued what would happen here if Godot finally showed up and joined the chat, and sadly, you'd probably have to write some code here to add commas between the names because right now you get "Chat between Vladimir and EstragonGodot".

Now that said, this is completely extra curricular stuff, so whatever.